src/Task/Builder/ChainedBuilder.php

```diff
@@ -21,13 +22,17 @@ final class ChainedBuilder extends AbstractTaskBuilder implements BuilderInterfa
      */
     private iterable $builders;
 
```

Comment:
Extra line to remove

Reply:
is it really an extra line as it is used as typehint

Reply:
Yes but we don't need it as we don't specify any informations on it 🙂

Reply:
I don't understand, which line is it exactly

Reply:
The extra blank line, number 24 🙂